arbnode/mel/delayed_message_backlog.go

```diff
@@ -18,40 +18,40 @@ type DelayedMeta struct {
 // created when that MSG was first SEEN
 //
 // This is to be initialized in the FetchInitialState function of InitialStateFetcher. If not, then Start fsm step of MEL runner initializes it
```

Comment:
This is a bit of an implementation detail and strange condition to have without good reason. A reviewer will probably be confused. Maybe we can brainstorm together how to better satisfy this invariant in code